mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndexTest.java

```diff
@@ -55,92 +55,92 @@ protected void initializeMailboxManager() {
         messageSearchIndex = resources.getSearchIndex();
     }
 
-    @Disabled("JAMES-1799")
+    @Disabled("JAMES-1799: ignoring failing test after port of ES tests to Lucene")
```

Comment:
`JAMES-1799: ignoring failing test after generalizing ElasticSearch test suite to other mailbox search backends` ?